storage/shard.go

```diff
@@ -893,6 +881,16 @@ func (s *dbShard) FlushState(blockStart time.Time) fileOpState {
 	return state
 }
 
+func (s *dbShard) markFlushStateSuccessOrError(blockStart time.Time, err error) error {
```

Comment:
Is there a reason why this should return the error?  Seems more idiomatic not to return err that was passed in without any modification.